Snake/BusinessLayer/AI.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+
+namespace BusinessLayer
+{
+    public class AI
+    {
+        private Direction currentDirection;
+        private int X;
+        private int Y;
+        Random randomNumber = new Random();
+        public int XCoordinate
+        {
+            get
+            {
+                return X;
+            }
+            set

```

Comment:
It turns out this setter is used once in the Engine class

 newAI.XCoordinate = previousX;
 newAI.YCoordinate = previousY;